### ISSUE TITLE
First step

### ISSUE THREAD

**Issue by ElGautinho** [ElGautinho/My-first-tutorial-project#2]
Explanation of the challenge, what your challenge consist of ?

**Comment by ElGautinho** [comment]
[README.md](https://github.com/user-attachments/files/16366542/README.md)

**Comment by ElGautinho** [comment]
<!DOCTYPE html>
<html lang="en">
    <link charset="UTF-8">
  <head>
    <title>Form Media</title>
    <link rel="stylesheet" href="styles.css" />
    <link rel="shortcut icon" href="logo.png">
  </head>
  <body>
    <div class="container">
      <header class="header">
        <h1 id="title" class="text-center">MEDIA LEPHARE-BUKAVU</h1>
        <p id="description" class="description text-center">Fill this form to be a member of this section</p>
      </header>
      <form id="survey">
        <div id="form-group"></div>
      </form>
    </div>
    </body>

**Comment by bahati7** [comment]
when you commit some code, please also refer to  the issue number concerned

**Comment by ElGautinho** [comment]
Yes i get it ... i'am about to correct that mistake by restarting the commit process of this project ... Then i'll be putting the issue number on every step of the code from the start to the end.
